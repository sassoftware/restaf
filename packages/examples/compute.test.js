let testFunctions = require("./testFunctions");
let testInfo;
beforeAll(async () => {
	testInfo = await require('./lib/setupAll')();
});
test("computeService: basic", async () => {
  expect.assertions();
  let l = await testFunctions.computeDS(testInfo);
  expect(l).toMatchSnapshot();
});
