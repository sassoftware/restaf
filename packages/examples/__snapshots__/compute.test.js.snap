// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`computeService: basic 1`] = `
Array [
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    1,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    2,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    3,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    4,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    5,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    6,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    7,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    8,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    9,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    10,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    11,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    12,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    13,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    14,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    15,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    16,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    17,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    18,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    19,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    20,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    21,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    22,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    23,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    24,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    25,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    26,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    27,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    28,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    29,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    30,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    31,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    32,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    33,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    34,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    35,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    36,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    37,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    38,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    39,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    40,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    41,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    42,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    43,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    44,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    45,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    46,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    47,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    48,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    49,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    50,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    51,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    52,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    53,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    54,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    55,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    56,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    57,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    58,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    59,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    60,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    61,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    62,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    63,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    64,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    65,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    66,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    67,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    68,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    69,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    70,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    71,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    72,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    73,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    74,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    75,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    76,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    77,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    78,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    79,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    80,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    81,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    82,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    83,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    84,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    85,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    86,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    87,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    88,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    89,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    90,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    91,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    92,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    93,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    94,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    95,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    96,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    97,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    98,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    99,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    100,
    11,
  ],
]
`;
